Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3109 @@
+{
```

Comment:
<ul><li>As you're not using it, you can silence the <code>p</code>  like that: <code>_, cov = np.polyfit</code>  </li><li>I think it should be <code>return np.mean([sim_coll(r) for _ in range(n)])</code> , otherwise you're not using <code>n</code> </li><li>I think "correlation" should be the x label. The y label is actually the posterior standard deviation of the slope relating F to K.</li><li>Also, I'm surprised because this standard deviation should <em>increase</em> with the correlation, not decrease. Actually, I think you're not regressing K on F and x, but you're just regressing it on F + x, hence the weird behavior we're getting. I'm guessing the equivalent of <code>lm</code> in python is <a href="https://www.statsmodels.org/stable/index.html" target="_blank">statsmodels</a>?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Good catch. It's been a long time since I looked at R symbolic formulas... I don't find them intuitive! I've switched to statsmodels.formula.api. It works, but is horribly slow. I'm not familiar enough with it to speed it up, but it produces a more sensible result now.</p>

Reply:
<p>Actually scratch that - using the formala.api made it <em>really</em> slow (took about 10x longer to run than the rest of the notebook!), so I switched to using scipy.optimize.curve_fit.</p>